lib/Basic/FileInfo.cpp

```diff
@@ -14,22 +14,93 @@
 
 #include <cassert>
 #include <cstring>
+
+#if defined(_WIN32)
+#include <windows.h>
+#include <direct.h>
+#else
 #include <sys/stat.h>
+#endif
 
 using namespace llbuild;
 using namespace llbuild::basic;
 
 bool FileInfo::isDirectory() const {
-  return mode & S_IFDIR;
+  return (mode & S_IFDIR) != 0;
 }
 
+#if defined(_WIN32)
+static void FileTimeToTimestamp(FILETIME fileTime, FileTimestamp *stamp) {
```

Comment:
The intent of FileInfo is not to present a normalized view of file time... I can see how it is nice to maintain a consistent reference point, but I'm not sure llbuild really should end up interpreting these times other than to compare them with one another.

In that case, we could simply decide to document that and not bother to put that conversion here. What do you think?